common/tdishell

```diff
@@ -0,0 +1,40 @@
+import socket
```

Comment:
It might be a better idea to use a name for this script that is not the same name as the p4sde/bin/tdishell